README.md

```diff
@@ -1,2 +1,25 @@
 # hw01
 Repository for homework 01
+
+#Homework Assignment 1
+## Jon Griffiths
+### Computing for the Social Sciences 
+### 8 April 2019
+
+**Generic Info**
+Hello everyone, my name is Jon Griffiths, and I'm a MAPSS student concentrating in American Politics. I mainly examine what factors increase or decrease voter registration and turnout rates. For my thesis, I'm analyzing the effects of the Automatic Voter Registration policies that have sprung up following the 2016 election in several states. If you wish to learn more about Automatic Voter Registration, check out this  [website](http://www.ncsl.org/research/elections-and-campaigns/automatic-voter-registration.aspx). If you're interested in learning about anything else, I strongly recommend this [website](https://www.google.com/).
+
+I'm from Cary, North Carolina but have spent the last 5 years Washington D.C. I've also done quite a bit of campaign work, so I've had extended stays in:
```

Comment:
My parents live in Wilmington, NC! Love getting away from the winter weather to go visit them.